examples/download_and_train.py

```diff
@@ -0,0 +1,97 @@
+import t4
+import os
+import json
+import pandas as pd
+
+
+from fnet.cli.init import save_default_train_options
+
+###################################################
+# Download the 3D multi-channel tiffs via Quilt/T4
+###################################################
+
+gpu_id = 0
+
+n_images_to_download = 40  # more images the better
+train_fraction = 0.75
+
+image_save_dir = '{}/images/'.format(os.getcwd())
+model_save_dir = '{}/model/'.format(os.getcwd())
+prefs_save_path = '{}/prefs.json'.format(os.getcwd())
+
+data_save_path_train = '{}/image_list_train.csv'.format(image_save_dir)
+data_save_path_test = '{}/image_list_test.csv'.format(image_save_dir)
+
+if not os.path.exists(image_save_dir):
+    os.makedirs(image_save_dir)
+
+
+aics_pipeline = t4.Package.browse(
+    "aics/pipeline",
+    registry="s3://quilt-aics"
+)
+
+image_ids = [k for k in aics_pipeline['fov']][:n_images_to_download]
+
+metadata = {}
+for image_id in image_ids:
+    metadata[image_id] = aics_pipeline['fov'][image_id].get_meta()
+
+image_save_paths = ['{}/{}'.format(image_save_dir, image_id) for image_id in image_ids]
+
+for image_id, image_save_path in zip(image_ids, image_save_paths):
+    if os.path.exists(image_save_path):
+        continue
+
+    # We only do this because T4 hates our filesystem. It probably wont affect you.
+    try:
+        aics_pipeline['fov'][image_id].fetch(image_save_path)
+    except OSError:
```

Comment:
The covered call is from a third-party package. It's hard to know what exactly can go wrong.

Reply:
I can kind of chime in here. It's not really our filesystem so much as this script was ran on the DGX. The DGX apparently has an older linux kernel and because of that a flag needs to be set for users to be allowed to store metadata on files. Something like `xattrs=True`, so this check is actually a decent method for handling when certain files have metadata attached and some not depending on filesystem / OS.